hare/hare.go

```diff
@@ -61,7 +62,7 @@ type Hare struct {
 
 	sign Signer
 
-	msh     meshProvider
+	mesh    meshProvider
```

Comment:
`msh` was used because `mesh` is a package name hence can be confusing.

Reply:
I see your point, but I'd rather optimize for readability. That package is not imported here, so there is no conflict, at least in this module. We could probably clean this up globally.